tests/fsharpqa/Source/Printing/ParamArrayInSignatures.fs

```diff
@@ -0,0 +1,11 @@
+// #Regression #NoMT #Printing 
```

Comment:
- I would rename this to .fsx since it's being run via FSI
- As is, piping this into FSI actually results in FSI exit code of 1, which causes the test to fail.
  - You need to add `#q` at the end to get a 0 exit code